### PR TITLE
Add column description for system tables

### PR DESCRIPTION
**Pull request by djshow832**
### What is changed, added or deleted? (Required)

Add column description for system tables `TIKV_REGION_STATUS` and `TIKV_STORE_STATUS`.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: system-tables/system-table-information-schema.md [hunk 1]
```diff
@@ -779,20 +779,48 @@ desc TIKV_REGION_STATUS;
...
+
+`TIKV_REGION_STATUS` 表中列的含义如下：
+
+* `REGION_ID`：Region 的 ID。
+* `START_KEY`：Region 的起始 key 的值。
+* `END_KEY`：Region 的末尾 key 的值。
+* `TABLE_ID`：Region 所属的表的 ID。
+* `DB_NAME`：`TABLE_ID` 所属的数据库的名称。
+* `TABLE_NAME`：Region 所属的表的名称。
+* `IS_INDEX`：Region 所属的表是否是索引，0 代表不是索引，1 代表是索引。
```
Comment (zimulala) [inline system-tables/system-table-information-schema.md:813]: Region 所属的表是否带索引数据？

File: system-tables/system-table-information-schema.md [hunk 2]
```diff
@@ -830,6 +858,28 @@ desc TIKV_STORE_STATUS;
...
 
+`TIKV_STORE_STATUS` 表中列的含义如下：
+
+* `STORE_ID`：Store 的 ID。
+* `ADDRESS`：Store 的地址。
+* `STORE_STATE`：Store 状态的标识符，与 `STORE_STATE_NAME` 相对应。
+* `STORE_STATE_NAME`：Store 状态的名字，为 `Up` / `Offline` / `Tombstone` 中的一种。
+* `LABEL`：给 Store 设置的标签。
+* `VERSION`：Store 的版本号。
+* `CAPACITY`：Store 的存储容量。
```
Comment (zimulala) [inline system-tables/system-table-information-schema.md:869]: 需要给个单位吗？

File: system-tables/system-table-information-schema.md [hunk 3]
```diff
@@ -830,6 +858,28 @@ desc TIKV_STORE_STATUS;
...
+* `AVAILABLE`：Store 的剩余存储空间。
+* `LEADER_COUNT`：Store 上的 leader 的数量。
+* `LEADER_WEIGHT`：Store 的 leader 权重。
+* `LEADER_SCORE`：Store 的 leader 评分。
+* `LEADER_SIZE`：Store 上的所有 leader 的大小。
```
Comment (zimulala) [inline system-tables/system-table-information-schema.md:874]: 所有 leader 容量大小？

File: system-tables/system-table-information-schema.md (hunk 2 above)
Comment (djshow832) [reply system-tables/system-table-information-schema.md:869]: The column itself has a unit, e.g. 49.98GiB

File: system-tables/system-table-information-schema.md [hunk 4]
```diff
@@ -779,20 +779,48 @@ desc TIKV_REGION_STATUS;
...
+
+`TIKV_REGION_STATUS` 表中列的含义如下：
+
+* `REGION_ID`：Region 的 ID。
+* `START_KEY`：Region 的起始 key 的值。
+* `END_KEY`：Region 的末尾 key 的值。
+* `TABLE_ID`：Region 所属的表的 ID。
+* `DB_NAME`：`TABLE_ID` 所属的数据库的名称。
+* `TABLE_NAME`：Region 所属的表的名称。
+* `IS_INDEX`：Region 数据是否是索引，0 代表不是索引，1 代表是索引。如果表和索引在同一个 region 上，会有多行记录，`IS_INDEX` 分别是 1 和 0。
```
Comment (zimulala) [inline system-tables/system-table-information-schema.md:813]: `如果表数据和索引数据都在当前的 region 上`？

File: system-tables/system-table-information-schema.md [hunk 5]
```diff
@@ -830,21 +830,49 @@ desc TIKV_REGION_PEERS;
...
-| START_KEY        | text                | YES  |     | <null>  |       |
-| END_KEY          | text                | YES  |     | <null>  |       |
-| EPOCH_CONF_VER   | bigint(21) unsigned | YES  |     | <null>  |       |
-| EPOCH_VERSION    | bigint(21) unsigned | YES  |     | <null>  |       |
-| WRITTEN_BYTES    | bigint(21) unsigned | YES  |     | <null>  |       |
-| READ_BYTES       | bigint(21) unsigned | YES  |     | <null>  |       |
-| APPROXIMATE_SIZE | bigint(21) unsigned | YES  |     | <null>  |       |
-| APPROXIMATE_KEYS | bigint(21) unsigned | YES  |     | <null>  |       |
-+------------------+---------------------+------+-----+---------+-------+
 ```
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:833]:
```suggestion
```sql
```

File: system-tables/system-table-information-schema.md [hunk 6]
```diff
@@ -830,21 +830,49 @@ desc TIKV_REGION_PEERS;
...
+
+`TIKV_REGION_STATUS` 表中列的含义如下：
+
+* `REGION_ID`：Region 的 ID。
+* `START_KEY`：Region 的起始 key 的值。
+* `END_KEY`：Region 的末尾 key 的值。
+* `TABLE_ID`：Region 所属的表的 ID。
+* `DB_NAME`：`TABLE_ID` 所属的数据库的名称。
+* `TABLE_NAME`：Region 所属的表的名称。
+* `IS_INDEX`：Region 数据是否是索引，0 代表不是索引，1 代表是索引。如果当前 region 同时包含表数据和索引数据，会有多行记录，`IS_INDEX` 分别是 1 和 0。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:865]:
```suggestion
* `IS_INDEX`：Region 数据是否是索引，0 代表不是索引，1 代表是索引。如果当前 Region 同时包含表数据和索引数据，会有多行记录，`IS_INDEX` 分别是 1 和 0。
```

File: system-tables/system-table-information-schema.md [hunk 7]
```diff
@@ -830,21 +830,49 @@ desc TIKV_REGION_PEERS;
...
+* `INDEX_ID`：Region 所属的索引的 ID。如果 `IS_INDEX` 为 0，这一列的值就为 NULL。
+* `INDEX_NAME`：Region 所属的索引的名称。如果 `IS_INDEX` 为 0，这一列的值就为 NULL。
+* `EPOCH_CONF_VER`：Region 的配置的版本号，在增加或减少 peer 时版本号会递增。
+* `EPOCH_VERSION`：Region 的当前版本号，在分裂或合并时版本号会递增。
+* `WRITTEN_BYTES`：已经往 Region 写入的数据量 (bytes)。
+* `READ_BYTES`：已经从 Region 读取的数据量 (bytes)。
+* `APPROXIMATE_SIZE`：Region 的近似数据量 (MB)。
+* `APPROXIMATE_KEYS`：Region 的近似 key 的数量。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:873]:
```suggestion
* `APPROXIMATE_KEYS`：Region 中 key 的近似数量。
```

File: system-tables/system-table-information-schema.md [hunk 8]
```diff
@@ -830,21 +830,49 @@ desc TIKV_REGION_PEERS;
...
+* `REPLICATIONSTATUS_STATE`：Region 当前的同步状态，为 `UNKNOWN` / `SIMPLE_MAJORITY` / `INTEGRITY_OVER_LABEL` 中的一种。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:874]:
```suggestion
* `REPLICATIONSTATUS_STATE`：Region 当前的同步状态，可能为 `UNKNOWN` / `SIMPLE_MAJORITY` / `INTEGRITY_OVER_LABEL` 其中一种状态。
```

File: system-tables/system-table-information-schema.md [hunk 9]
```diff
@@ -882,6 +910,28 @@ desc TIKV_STORE_STATUS;
...
+* `STORE_STATE_NAME`：Store 状态的名字，为 `Up` / `Offline` / `Tombstone` 中的一种。
+* `LABEL`：给 Store 设置的标签。
+* `VERSION`：Store 的版本号。
+* `CAPACITY`：Store 的存储容量。
+* `AVAILABLE`：Store 的剩余存储空间。
+* `LEADER_COUNT`：Store 上的 leader 的数量。
+* `LEADER_WEIGHT`：Store 的 leader 权重。
+* `LEADER_SCORE`：Store 的 leader 评分。
+* `LEADER_SIZE`：Store 上的所有 leader 的近似总数据量 (MB)。
+* `REGION_COUNT`：Store 上的 region 总数。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:927]:
```suggestion
* `REGION_COUNT`：Store 上的 Region 总数。
```

File: system-tables/system-table-information-schema.md [hunk 10]
```diff
@@ -882,6 +910,28 @@ desc TIKV_STORE_STATUS;
...
+* `REGION_WEIGHT`：Store 的 region 权重。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:928]: ditto

File: system-tables/system-table-information-schema.md [hunk 11]
```diff
@@ -882,6 +910,28 @@ desc TIKV_STORE_STATUS;
...
+* `REGION_SCORE`：Store 的 region 评分。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:929]: ditto

File: system-tables/system-table-information-schema.md [hunk 12]
```diff
@@ -882,6 +910,28 @@ desc TIKV_STORE_STATUS;
...
+* `REGION_SIZE`：Store 上的所有 region 的近似总数据量 (MB)。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:930]: ditto

File: system-tables/system-table-information-schema.md [hunk 13]
```diff
@@ -876,20 +876,48 @@ desc TIKV_REGION_STATUS;
...
+
+`TIKV_REGION_STATUS` 表中列的含义如下：
+
+* `REGION_ID`：Region 的 ID。
+* `START_KEY`：Region 的起始 key 的值。
+* `END_KEY`：Region 的末尾 key 的值。
+* `TABLE_ID`：Region 所属的表的 ID。
+* `DB_NAME`：`TABLE_ID` 所属的数据库的名称。
+* `TABLE_NAME`：Region 所属的表的名称。
+* `IS_INDEX`：Region 数据是否是索引，0 代表不是索引，1 代表是索引。如果当前 Region 同时包含表数据和索引数据，会有多行记录，`IS_INDEX` 分别是 1 和 0。
```
Comment (yikeke) [inline system-tables/system-table-information-schema.md:910]: 「`IS_INDEX` 分别是 1 和 0」-> 「`IS_INDEX` 分别是 0 和 1」？

表数据对应 0，索引数据对应 1？ @djshow832

Comment (djshow832) [reply system-tables/system-table-information-schema.md:910]: 对

Comment (yikeke) [reply system-tables/system-table-information-schema.md:910]: 好的，我一起改下吧

**Pull request merged**